### DISCUSSION TITLE
Sharing code in mediatr command handlers

### DISCUSSION THREAD

**Discussion by kreditkenny** [LuckyPennySoftware/MediatR#770] [General]
this has been a challenge and wondering how folks here have been sharing code, specifically in command handlers? examples are welcomed.

**Comment by pawel87** [comment]
You can write base class where you will have your shared code.
In that case you can create several handlers that will inherit from this base class.

Another alternative it to move your shared code in some reusable components, like helpers/builders...depending on your needs.
